README.md

```diff
@@ -1,14 +1,13 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
+![screenshot](images/Capture%20d%E2%80%99%C3%A9cran%202022-05-17%20%C3%A0%2017.55.24.png)
 
 <br />
 
-# Awesome Book
+# CUBIS - CAPSTONE PROJECT
 
-> This app is to manage todo list
+> CUBIS FOOD is an application that allows you to find meal ideas that you can make simply with the ingredients you have.
 
 ## LOOM VIDEO PRESENTING
-[Watch the presentation of the projet on Loom](#)
+[Watch the presentation of the projet on Zoom](https://drive.google.com/file/d/1X4PMdm3tavYzzbfjhfqmQocGPMrYIYLB/view?usp=sharing)
```

Comment:
- [ ] very nice presentation of your project however in the video all the students who worked on the project should do a part of the presentation. in addition to that, each student should present a piece of code they are proud of.
- [ ] [OPTIONAL] It would be nice and make the video look a lot more professional if all students have their cameras on.